bootstrap.sh

```diff
@@ -19,39 +46,89 @@ then
+cd openfisca-$NO_SPACES_JURISDICTION_LABEL
 
 git init
 git add .
-git commit --no-gpg-sign --message 'Initial import from OpenFisca country-template' --author='OpenFisca Bot <bot@openfisca.org>'
+git symbolic-ref HEAD refs/heads/main
+git commit --no-gpg-sign --message "$first_commit_message" --author='OpenFisca Bot <bot@openfisca.org>'
+echo -e "${YELLOW}--\033[0m"
+echo -e "${PURPLE}Set default branch to 'main'.\033[0m"
+echo -e "${PURPLE}Initial git commit made to 'main' branch: \033[0m${BLUE}$first_commit_message\033[0m"
+echo -e "${YELLOW}--\033[0m"
 
 all_module_files=`find openfisca_country_template -type f ! -name "*.DS_Store"`
 
 set -x
 
+
 # Use intermediate backup files (`-i`) with a weird syntax due to lack of portable 'no backup' option. See https://stackoverflow.com/q/5694228/594053.
-sed -i.template "s|country_template|$lowercase_country_name|g" README.md setup.py setup.cfg .github/workflows/workflow.yml Makefile MANIFEST.in $all_module_files
-sed -i.template "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py .github/workflows/workflow.yml .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md $all_module_files
+sed -i.template "s|openfisca-country_template|openfisca-$NO_SPACES_JURISDICTION_LABEL|g" README.md Makefile pyproject.toml CONTRIBUTING.md
+sed -i.template "s|country_template|$CODE_JURISDICTION_LABEL|g" README.md pyproject.toml .flake8 .github/workflows/workflow.yml Makefile MANIFEST.in $all_module_files
+sed -i.template "s|Country-Template|$JURISDICTION_NAME|g" README.md pyproject.toml .github/workflows/workflow.yml .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md
 sed -i.template -e "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
-sed -i.template "s|country-template|$lowercase_country_name|g" README.md
-sed -i.template "s|https://github.com/openfisca/openfisca-country-template|$URL|g" setup.py
+sed -i.template "s|https://example.com/repository|$REPOSITORY_URL|g" README.md CONTRIBUTING.md
+sed -i.template -e "1,${last_changelog_number}d" CHANGELOG.md  # remove country-template CHANGELOG leaving example
+sed -i.template "s|https://github.com/openfisca/country-template|$REPOSITORY_URL|g" pyproject.toml
+sed -i.template "s|repository_folder|$REPOSITORY_FOLDER|g" README.md
 find . -name "*.template" -type f -delete
 
 set +x
 
-git mv openfisca_country_template openfisca_$lowercase_country_name
+git mv openfisca_country_template openfisca_$CODE_JURISDICTION_LABEL
 
 git rm bootstrap.sh
 git add .
-git commit --no-gpg-sign --message 'Customise country-template through script' --author='OpenFisca Bot <bot@openfisca.org>'
+git commit --no-gpg-sign --message "$second_commit_message" --author='OpenFisca Bot <bot@openfisca.org>'
+echo -e "${YELLOW}--\033[0m"
+echo -e "${PURPLE}Second git commit made: \033[0m${BLUE}$second_commit_message\033[0m"
 git remote add origin $REPOSITORY_URL.git
+echo -e "${PURPLE}Git remote origin added: \033[0m${BLUE}$REPOSITORY_URL.git\033[0m"
+echo -e "${YELLOW}--\033[0m"
```

Comment:
> I feel like a lot of this verbosity could be replaced with `set -x`.

Reply:
Have clarified both the README.md and the bootstrap.sh file ([600f986](https://github.com/openfisca/country-template/pull/139/commits/600f9869bde7cdd00918f769c244b5b0d81136ae)) to further improve the output and reduce this verbosity.